docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Machine ID Getting Started Guide
+description: Getting started with Teleport Machine ID
+---
+
+# Getting Started
+
+In this getting started guide we will use Machine ID to create an identity for a machine and use that identity to connect to a SSH server.
+
+Here's an overview of what we will do:
+
+1. Download and install Teleport (=teleport.version=).
+2. Create Bot User
+3. Start Machine ID
+4. Use Machine ID
+
+## Prerequisites
+
+Before using Machine ID, you will need an existing Teleport cluster.
+
+If you have not setup a Teleport cluster before, follow the [Getting started on a Linux Server](https://goteleport.com/docs/getting-started/linux-server/) guide for instructions.
+
+## Step 1/4. Download and install Teleport (=teleport.version=).
+
+Machine ID is available starting from `9.0.0` release.
+
+Download the appropriate version of Teleport for your platform from
+our [downloads page](https://goteleport.com/teleport/download).
+
+You will be using the `tbot` binary from the package you download and install.
+
+## Step 2/4. Create Bot User
+
+Before you create a bot user, you need to determine which role you want to assign to it. You can use `tctl` to examine what roles exist on your system.
+
+```bash
```

Comment:
I think we should use `code` here so we can take advantage of our in-house formatting and logic.